airflow/www/static/js/grid/details/content/dagRun/index.jsx

```diff
@@ -27,10 +27,10 @@ import {
 } from '@chakra-ui/react';
 import { MdPlayArrow, MdOutlineAccountTree } from 'react-icons/md';
 
-import { SimpleStatus } from '../../../StatusBox';
-import { ClipboardText } from '../../../Clipboard';
+import { SimpleStatus } from '../../../components/StatusBox';
+import { ClipboardText } from '../../../components/Clipboard';
```

Comment:
Is there a way to do absolute import from source root of the grid app to avoid doing multiple `../`

I don't know where we want to set the limit with relative imports but I think it might be easier to maintain in such cases. (not related to this PR particularly, just being curious)

Reply:
I agree, absolute paths has been on my list for a while now.